pom.xml

```diff
@@ -293,6 +293,12 @@
                     <configuration>
                         <argLine>${argLine} -Duser.timezone=UTC -Dddf.version=${ddf.version}
                         </argLine>
+                        <includes>
+                            <include>**/*Test.java</include>
+                            <include>**/Test*.java</include>
+                            <include>**/*TestCase.java</include>
+                            <include>**/*Spec.class</include>
```

Comment:
Where do we use `*Spec.class`? Should we include groovy unit test regexs here too?